web/app/src/App.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import {createRoot} from "react-dom/client";
+
+const container = document.getElementById('app');
+
+function App() {
+    return (<div>
+        Project init manala
+    </div>)
+}
+
+createRoot(container).render(<App />);
```

Comment:
What about giving a try to the React `StrictMode` ?

https://react.dev/reference/react/StrictMode